src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -44,7 +44,7 @@ private Institution createInstitution(Integer userId) {
   }
 
   private User createUserForTest() {
-    Integer userId = userDAO.insertUser(RandomStringUtils.random(10), RandomStringUtils.randomAlphabetic(10), new Date());
+    Integer userId = userDAO.insertUser(RandomStringUtils.randomAlphabetic(10), RandomStringUtils.randomAlphabetic(10), new Date());
```

Comment:
See example error case from this test method: https://github.com/DataBiosphere/consent/runs/8277876664?check_suite_focus=true

```
org.jdbi.v3.core.statement.UnableToExecuteStatementException: org.postgresql.util.PSQLException: 
ERROR: invalid byte sequence for encoding "UTF8": 0x00 
[statement:"/* UserDAO.insertUser */ 
    INSERT INTO users (email, display_name, create_date) values (:email, :displayName, :createDate)", 
    arguments:{positional:{0:콑鉲𓋩🏐𒔏瑲,1:kXMWMacZVu,2:Fri Sep 09 20:12:49 UTC 2022}, named:{displayName:kXMWMacZVu,email:콑鉲𓋩🏐𒔏瑲,createDate:Fri Sep 09 20:12:49 UTC 2022}, finder:[]}]
```